doc/lua_api.txt

```diff
@@ -554,9 +554,11 @@ node definition:
     ^ The rotation of the node is stored in param2. Furnaces and chests are
       rotated this way. Can be made by using minetest.dir_to_facedir().
       Values range 0 - 23
-      facedir modulo 4 = axisdir
+      facedir / 4 = axis direction:
       0 = y+    1 = z+    2 = z-    3 = x+    4 = x-    5 = y-
-      facedir's two less significant bits are rotation around the axis
+      facedir modulo 4 = rotation around that axis
+      facedir 0, 1, 2, 3 = rotations around the +y axis
```

Comment:
This is only an example for the general rule explained by lines 557, 558 and 559.